runbooks/OrphanedVirtualMachineImages.md

```diff
@@ -0,0 +1,77 @@
+# OrphanedVirtualMachineImages
+
+## Meaning
+
+This alert fires when a VMI (virt-launcher-* pod) is running on a node that does not have a running `virt-handler-*` pod.  
+
+## Impact
+
+When a(n) VMI(s) is running on a node that does not have a running `virt-handler-*` those VMIs are considered orphaned. Meaning that they are no longer able to be managed.
```

Comment:
You started with singular (_is running_) and ended with plural (_those VMIs_). I suggest to keep with same all along the sentence.
are considered orphaned --> are considered *as* orphaned.
no longer able to be managed --> no longer manageable